### PR TITLE
Add tokenizer to bleu

### PR DESCRIPTION
**Pull request by szhengac**
## Description ##
This pr adds tokenizer to bleu script for plain text inputs. This can produce the official bleu score in WMT. The script integrates the tokenizer in https://github.com/awslabs/sockeye/blob/master/contrib/sacrebleu/sacrebleu.py 

### Essentials ###
- [] Changes are complete (i.e. I finished coding on this PR)
- [] All changes have test coverage
- [] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-154/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-154/5/index.html

File: scripts/nmt/bleu.py [hunk 1]
```diff
@@ -75,16 +78,110 @@ def _bpe_to_words(sentence, delimiter='@@'):
...
+    norm = re.sub(r'([\.,])([^0-9])', ' \\1 \\2', norm)
+    norm = re.sub(r'([0-9])(-)', '\\1 \\2 ', norm)
+    norm = re.sub(r'\s+', ' ', norm)
+    norm = re.sub(r'^\s+', '', norm)
+    norm = re.sub(r'\s+$', '', norm)
+
+    return norm
+
+
+class UnicodeRegex: #pylint: disable=old-style-class
```
Comment (szha) [inline scripts/nmt/bleu.py]: why not inherit object

**Comment by mli** [comment]
Job PR-154/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-154/7/index.html

**Comment by mli** [comment]
Job PR-154/11 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-154/11/index.html

**Pull request merged**